statement-summary-tables.md

```diff
@@ -79,8 +83,120 @@ The following is a sample output of querying `events_statements_summary_by_diges
+
+## Parameter configuration
+
+Use the following system variables to control the statement summary:
+
+- `tidb_enable_stmt_summary`: Determines whether the statement summary feature is enabled. `1` represents `enable`, `0` means `disable`. The feature is enabled by default. The statistics in the system table are cleared if this feature is disabled. The statistics are re-calculated next time the feature is enabled. Tests have shown that enabling this feature has little impact on performance.
+- `tidb_stmt_summary_refresh_interval`: The interval at which the `statements_summary` table is refreshed. The time unit is second (s). The default value is `1800`.
+- `tidb_stmt_summary_history_size`: The size of each SQL statement category stored in the `statements_summary_history` table. The default value is `24`.
+- `tidb_stmt_summary_max_stmt_count`: Limits the number of SQL statements that can be stored in statement summary tables. The default value is `200`. If the limit is exceeded, those SQL statements that recently remain unused are cleared.
+- `tidb_stmt_summary_max_sql_length`: Specifies the longest display length of `DIGEST_TEXT` and `QUERY_SAMPLE_TEXT`. The default value is `4096`.
+- `tidb_stmt_summary_internal_query`: Determines whether to count the TiDB SQL statements. `1` means `count`, 0 means `not count`. The default value is `0`.
+
+The following shows a configuration example:
+
+{{< copyable "sql" >}}
+
+```sql
+set global tidb_enable_stmt_summary = true;
+set global tidb_stmt_summary_refresh_interval = 1800;
+set global tidb_stmt_summary_history_size = 24;
+```
+
+When the above configuration takes effect, every 30 minutes the `statements_summary` is cleared. `statements_summary_history` stores data generated over the recent 12 hours.
+
+The above system variables have two scopes: global and session. These scopes work differently from other system variables:
+
+- Set the global variable to apply to the cluster immediately.
+- Set the session variable to apply to the current TiDB server immediately. This is useful when you debug on a single TiDB server instance.
+- The session variable has a higher read priority. The global variable is read only when no session variable is set.
+- Set the session variable to a blank string to re-read the global variable.
+
+> **Note:**
+>
+> The `tidb_stmt_summary_history_size`, `tidb_stmt_summary_max_stmt_count`, and `tidb_stmt_summary_max_sql_length` configuration items affect memory usage. It is recommended that you adjust these configurations based on your needs. It is not recommended to set them too large values.
+
+## Limitations
+
+The statement summary tables have the following limitations:
+
+The statement summary data are lost when the TiDB server is restarted. This is because statement summary tables are memory tables, and the data is cached in memory instead of being persisted on storage.
```

Comment:
```suggestion
The statement summary data will be lost when the TiDB server is restarted. This is because statement summary tables are memory tables, and the data is cached in memory instead of being persisted on storage.
```